docs/config/conf.py

```diff
@@ -22,8 +24,8 @@
 author = "Intel"
 
 # The full version, including alpha/beta/rc tags
-release = "0.11.0"
-
+release = os.getenv("UMF_VERSION", "unknown")
```

Comment:
pls update docs/README if needed

Reply:
done

Reply:
I meant `docs/README` file 😉  buuut, I just realized we now have `make docs` target - so you could set proper env variable within the target creation, so it'd work both locally and in CI (without the extra step in CI)

Reply:
done